src/pages/admin/Companion/index.js

```diff
@@ -128,17 +130,18 @@ const Companion = () => {
   };
 
   const createQR = async () => {
-    const keys = Object.keys(errors);
     const newErrors = {
       ...errors
     };
-    let validation = true;
-    keys.forEach((key) => {
-      if (newQR[key] === "") {
-        validation = false;
-        newErrors[key] = true;
-      }
-    });
+    const validation = true;
+    // @TODO readd validation
```

Comment:
it seems the validations are broken with both partner and workshop info, some refactoring needs to be done so we can add stuff in the future without screwing with this, if you have time @AllanT102 today to implement it lmk, or else Im gonna disable it and make a ticket for the future